Framework/DataHandling/src/BankPulseTimes.cpp

```diff
@@ -25,7 +26,14 @@ BankPulseTimes::BankPulseTimes(::NeXus::File &file,
     : periodNumbers(pNumbers) {
   file.openData("event_time_zero");
   // Read the offset (time zero)
-  file.getAttr("offset", startTime);
+  if (file.hasAttr("offset"))
+    file.getAttr("offset", startTime);
+  else if (file.hasAttr("start"))
+    file.getAttr("start", startTime);
+  else {
```

Comment:
This only makes sense if you know that the ESS doens't record offset or start and uses a unix_epoch. Could you please add comment about where this comes from to help future maintainers?